src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -1138,3 +1163,69 @@ export const getStatus = (response) => {
     ? response.errors[0].status
     : 'unknown';
 };
+
+// Series of utility functions to sort claims and appeals by last updated date
+/**
+ * 
+ * @param {Object} appeal
+ * @returns {string}
+ */
+const getAppealDate = (appeal) => {
+  const { events } = appeal.attributes;
+  const dateString = (events && events.length)
+    ? events[events.length - 1].date
+    : '0';
+  return dateString;
+};
+
+/**
+ * 
+ * @param {Object} claim
+ * @returns {string}
+ */
+const getClaimDate = (claim) => {
+  const { phaseChangeDate } = claim.attributes;
+  const dateString = phaseChangeDate || '0';
+  return dateString;
+};
+
+/**
+ * 
+ * @param {Object} item
+ * @returns {string}
+ */
+const getDate = (item) => {
+  if (!item.attributes) {
+    return '0';
```

Comment:
Using this, it may end up at 1970-01-01. To ensure this ends up at the bottom of the pile, what do you think of returning the date that VA was founded?